pelicanconf.py

```diff
@@ -39,6 +39,7 @@
 DISPLAY_PAGES_ON_MENU = False
 DISPLAY_CATEGORIES_ON_MENU = False
 MENUITEMS = (
+    ('&laquo; Register &raquo;', 'https://sciwork.kktix.cc/events/sciwork2020'),
```

Comment:
Are `&laquo` and `&raquo` a good idea?

Reply:
I've update the register button style to make looks better :)